.golangci.yml

```diff
@@ -1,6 +1,6 @@
 run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
-  timeout: 2m
+  timeout: 5m
```

Comment:
Leave this at `2m` I'll re-run any CI jobs/tests that fail due to timeout issues.